ruby/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM ubuntu:20.04
+RUN apt-get update && apt-get install -y ruby-full
+CMD ["tail", "-f", "/dev/null"]
```

以下のコードで動きませんか？
（実際に動かしてないので動かなかったらすみません🙇‍♂️）

```suggestion
FROM ruby:3.3
```

---

上記のコードでも動きました！
僕もDockerHubの公式のものからビルドしようと思ったのですが
ソースみた感じubuntuではなくDebianというディストリビューションだったため、ubuntuをベースにrubyをインストールするようにした方が良いのかと勝手に認識してしまっておりました💦

ubuntuはDebianをベースにしたものだから、同じものという認識で良かったですか？

この辺の知識あまりないので的外れなこと言ってたらすみません💦

---

@rikuya98 
すみません、これは私が間違ってました 💦 （私のミスリードです。）
`FROM ruby:3.3`にすると、rikuyaさんがおっしゃる通りディストビューションがDebianになってしまいます。
今回、特に強い理由でubuntuを使用したい訳ではないので、debianでお願いしても良いでしょうか？

https://ja.wikipedia.org/wiki/Linux%E3%83%87%E3%82%A3%E3%82%B9%E3%83%88%E3%83%AA%E3%83%93%E3%83%A5%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3

---

ちなみに、ubuntuイメージでやるなら、以下のようにする必要がありますね。
（rubyのbaseイメージでubuntuが存在しないため。）
Ref: https://hub.docker.com/_/ruby

```dockerfile
FROM ubuntu:latest

RUN apt-get update -y && apt-get install -y ruby-full

RUN gem install ruby:3.3.0
```

---

そうだったのですね！
ではシンプルにrubyの公式のベースイメージからbuild→マウント→rubyプログラム実行
という流れでよかったということですね！
ありがとうございます！

latestをつけることで確実に最新版を取得できるのですね
gem installとしてる部分でrubyのverを指定？固定？するという感じですかね？

---

@rikuya98 
ごめんなさい、`RUN gem install ruby:3.3.0`は要らなかったです！
versionを固定したいなら、rbenv等を使わないといけなさそうです！